chain/indexer/src/streamer/mod.rs

```diff
@@ -353,9 +401,9 @@ pub(crate) async fn start(
 
                 match response {
                     Ok(streamer_message) => {
-                        debug!(target: INDEXER, "{:#?}", &streamer_message);
```

Comment:
Trust me, the entire `StreamerMessage` in the logs does not help to debug the indexer.